index.html

```diff
@@ -80,7 +80,7 @@ <h3 class="text-muted">Chromebrew</h3>
       </div>
 
       <div class="row marketing">
-        <div class="col-lg-6">
+        <div class="col-6">
```

Comment:
Can you remove this line and https://github.com/flyingP0tat0/chromebrew/blob/5b00fb210cf93e95ca08305b9cf6a3c24e1e4bf7/index.html#L91?